zonemaster/zonemaster-gui
# Updates dependencies for v2024.2 (v4.3.1)

## Purpose

This PR updates dependencies (`package.json`) to reduces warnings when building. It also includes the resulting `package-lock.json` to cache the result of `npm audit fix`.

This PR is built on top of  commit 2c299b36fc956b26b1086d9ee8d26ddeb8d40707 in #484, and the files from that are not included in this PR.

## Changes

* package.json
* package-lock.json

## How to test this PR

Verify that resulting package runs well. Use the zip file enclosed in the comment below.
